### DISCUSSION TITLE
Installation already possible?

### DISCUSSION THREAD

**Discussion by simplyzetax** [jeremylvln/Shulker#91] [Q&A] [answered]
Are we already able to use and install this system? Because the documentation is still unfinished. Seems like a really cool project and i'd be interested in giving it a try

**Comment by jeremylvln** [comment, marked as answer]
Hey! I never tested the real installation until last week. Now that I have a working "well installed" setup, I plan to update the documentation and create the first alpha release really soon :)

**Reply by jeremylvln** [reply]
I've written the Getting Started chapters yesterday and made the first release :) Nothing is quite perfect but it should be sufficient for a first alpha release!
